tests/unit/config/test_api.py

```diff
@@ -23,11 +22,3 @@ def test_default_value_in_data(self):
             'test_default_value',
             "The value of config_variable.data['default_value'] should be the same "
             "as set as second argument of ConfigVariable()")
-
-
-class TestConfigHandler(TestCase):
```

Comment:
This test should not be deleted. It should also work and test that `config['unknown']` leads to the specific error in server code.